upgrade/helpers/tasks.py

```diff
@@ -998,3 +998,21 @@ def upgrade_validation(upgrade_type=False):
     if upgrade_type:
         run('hammer ping', warn_only=True)
     run('katello-service status', warn_only=True)
+
+
+def mongo_db_engine_upgrade(upgrade_type):
+    """
+    The purpose of this method to perform the upgrade of mongo DB database engine
+    from MMAPv1 to WiredTiger.
+    :param str upgrade_type: upgrade type would be a string, depends on choice If user
+    select the upgrade_type 'Satellite' then mongodb upgrade would be performed on
+    Satellite otherwise it would be happened on Capsule
+    """
+    logger.highlight('\n========== MongoDB DataBase Engine Upgrade =================\n')
+    logger.info("Upgrading the MongoDb Database on {}".format(upgrade_type))
+    preup_time = datetime.now().replace(microsecond=0)
+    run("satellite-installer --upgrade-mongo-storage-engine")
```

Comment:
How about replacing this with foreman-maintain upgrading mongo db, if it supports ? Inputs - @ntkathole @jameerpathan111 .

Reply:
@jyejare we don't have any option for it in f-m as of now.

Reply:
@ntkathole Thanks for confirmation. @devendra104 Lets keep that line as it is.